src/agent.spec.ts

```diff
@@ -1,74 +1,262 @@
-import {
-  FindingType,
-  FindingSeverity,
-  Finding,
-  HandleTransaction,
-  createTransactionEvent,
-  ethers,
-} from "forta-agent";
-import agent, {
-  ERC20_TRANSFER_EVENT,
-  TETHER_ADDRESS,
-  TETHER_DECIMALS,
-} from "./agent";
-
-describe("high tether transfer agent", () => {
+import { FindingType, FindingSeverity, HandleTransaction } from "forta-agent";
+import { Interface } from "ethers/lib/utils";
+import { createAddress } from "forta-agent-tools";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { provideHandleTransaction } from "./agent";
+import { BigNumber } from "ethers";
+import { methods } from "./constants";
+
+describe("Nethermind bot deployment to Forta Bot Registry", () => {
```

Comment:
Good test coverage! Would just request to add cases when the deployer account is NOT the Nethermind account, to make sure those don't get flagged, with all of its variations like you have for the Nethermind account.